mule-packager/src/main/java/org/mule/tools/api/util/MavenProjectInformation.java

```diff
@@ -167,7 +170,8 @@ public Pom getEffectivePom() {
                                                                                     Settings settings,
                                                                                     List<String> customDomains) {
     ExchangeRepositoryMetadata metadata = null;
-    if (repository != null && new ExchangeUriChecker(customDomains).isExchangeRepo(repository.getUrl())) {
+    ExchangeUriChecker exchangeUriChecker = new ExchangeUriChecker(customDomains);
+    if (repository != null &&exchangeUriChecker.isExchangeRepo(repository.getUrl())) {
```

Comment:
if you need to commit something else, place a space between & and the second clause

Reply:
🤦